cmd/node/main.go

```diff
@@ -1090,20 +1090,12 @@ func createNode(
 		if err != nil {
 			return nil, err
 		}
-		err = nd.CreateShardGenesisBlock()
```

Comment:
delete unused functions from node.go

Reply:
Done. Also a test was needed to be removed.